src/crud.js

```diff
@@ -0,0 +1,35 @@
+const saveLocalStorage = (arr) => {
+  localStorage.setItem('toDoList', JSON.stringify(arr));
+};
+
+const loadLocalStorage = (arr) => {
+  const localToDoList = JSON.parse(localStorage.getItem('toDoList'));
+  if (localToDoList) arr = localToDoList;
+  return arr;
+};
+
+const addTask = (arr, task) => {
+  arr.push({ id: arr.length + 1, description: task, completed: false });
+  saveLocalStorage(arr);
+};
```

Comment:
- [ ] Take a look at the image below. Your tasks are missing the `index` property as stated in the requirements. Kindly add this property.

  ![gaby-index](https://user-images.githubusercontent.com/36542986/149259613-03fe575b-5216-4f3d-b00e-9b425a9bc521.jpg)